recipes/ocw/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ocw" %}
+{% set version = "1.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "8f12405ec02ea327b1e0cf65547e409232f67577be6eb348d2ea9011827e2c4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
```

Comment:
Please add the arguments ` --single-version-externally-managed --record record.txt`. These are required when using `setuptools` to make sure this gets packaged correctly.